deploy/crds/mongodb.com_mongodbs_crd.yaml

```diff
@@ -31,19 +31,18 @@ spec:
         spec:
           description: MongoDBSpec defines the desired state of MongoDB
           properties:
-            kind:
-              description: Kind defines which kind of MongoDB deployment the resource
-                should create
-              type: string
```

Comment:
is this a manual edit or automatically generated?

![](https://media.giphy.com/media/NEJCPYzr6xmAU/giphy.gif)

Reply:
automatically generated with `operator-sdk generate crds`